submodules/usersCallForwarding/views/complexStrategy.html

```diff
@@ -0,0 +1,184 @@
+                {{/monsterRadio}}
+
+                <div>
+                    {{#monsterRadio @root.i18n.users.callForwarding.strategies.selective.specific }}
+                    <input type="radio" class="radio-state" name="{{strategy}}.from" value="specific" {{#compare
+                        type '===' 'specific' }} checked{{/compare}} />
+                    {{/monsterRadio}}
+
+                    <div class="selective-control-group specific disabled">
+                        <div class="specific-phone-number-wrapper">
+                            <label for="{{strategy}}.phoneNumber.value" class="control-label">
+                                {{@root.i18n.users.callForwarding.strategies.common.phoneNumber.inputLabel}}
+                            </label>
+                            <div class="controls specific-controls">
+                                <input class="controlls-width margins" name="{{strategy}}.phoneNumbers[]" type="phoneNumber" value="{{user.call_forward.selective.number}}">
+                                <a class="remove-button">X</a>
+                            </div>
+                        </div>
+                        <a class="add-phone-number">+ Number</a>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+
+        <div class="forward-to">
+            <label>
+                Forward calls to
+            </label>
+
+            <div class="complex-rule">
+                <div class="voicemail-wrapper">
+                    <div>
+                        {{#monsterRadio @root.i18n.users.callForwarding.strategies.common.voicemail.label }}
+                        <input type="radio" class="radio-state" name="{{strategy}}.type" value="voicemail" {{#compare
+                            type '===' 'voicemail' }} checked{{/compare}} />
+                        {{/monsterRadio}}
+                    </div>
+                    <div>
+                        <div class="selective-control-group voicemail">
+                            <label class="control-label" for="{{strategy}}.voicemail.value">
+                                {{@root.i18n.users.callForwarding.strategies.common.voicemail.selectLabel}}
+                            </label>
+                            <div class="controls specific-controls">
+                                <select class="controlls-width" id="{{strategy}}_voicemail_select"
+                                    name="{{strategy}}.voicemail.value">
+                                    {{#select selectedVoicemailId}}
+                                    {{#each voicemails as |voicemail|}}
+                                    <option value="{{voicemail.id}}">
+                                        {{voicemail.name}}
+                                    </option>
+                                    {{/each}}
+                                    {{/select}}
```

Comment:
Here it's just the spacing.

```suggestion
                                    {{#select selectedVoicemailId}}
                                          {{#each voicemails as |voicemail|}}
                                          <option value="{{voicemail.id}}">
                                              {{voicemail.name}}
                                          </option>
                                          {{/each}}
                                    {{/select}}
```